### DISCUSSION TITLE
Signup problems

### DISCUSSION THREAD

**Discussion by aapnetin** [outline/outline#3529] [General] [answered]
Outline only allows signups from Google, Microsoft and Slack business accounts only ? !!

![image](https://user-images.githubusercontent.com/103986280/168419137-01c41cee-5f99-48c9-a6a9-c2fa32e8543c.png)

**Comment by Limezy** [comment, marked as answer]
- If you are talking about the commercial SAAS getoutline.com, I guess the answer is yes.
- If you are talking about the Open Source self-hostable software, you can use any OIDC compatible provider (or develop your own passport.js auth connector if needed)
